src/scss/layout/_footer.scss

```diff
@@ -0,0 +1,48 @@
+.footer {
+  background: #000;
```

Comment:
Replace the colors with the right variables defined in the file `_variables.scss`

Reply:
Done. Replace the colors for the variables defined in the fie _variables.scss